### PR TITLE
 CASH-763: Styling updates for Filter Menu

### PR DESCRIPTION
**Pull request by ksekhri**

### CODE REVIEW

File: src/components/Kv/KvCheckbox.vue [hunk 1]
```diff
@@ -1,24 +1,22 @@
...
-				type="checkbox"
-				v-model="checkboxStatus"
-				@change="handleChange"
-				@input="handleInput"
-			>
-			<slot v-if="hasDefaultSlot"></slot>
-			<span v-else-if="label">{{ label }}</span>
-		</label>
-	</div>
+	<label class="kv-checkbox">
```
Comment (ksekhri) [inline src/components/Kv/KvCheckbox.vue]: Figured I'd unwrap this

File: src/pages/Lend/Filter/FilterMenuSection.vue [hunk 2]
```diff
@@ -1,19 +1,25 @@
...
+			<div class="filter-summary-title-row">
+				<div class="filter-summary-title" >
+					{{ title }}
+				</div>
+				<kv-icon class="filter-summary-title-chevron" name="small-chevron-mobile" />
 			</div>
 			<div class="filter-summary-applied-filters">{{ appliedFilters }}</div>
 		</div>
-		<div class="filter-items-container">
+		<div class="filter-items-container" @click="handleClickFilterItemsContainer">
```
Comment (ksekhri) [inline src/pages/Lend/Filter/FilterMenuSection.vue]: Basically I increased the click area for toggling the filter to include the padding. This handler adds a `stopPropagation` so clicking on filters themselves doesn't toggle the menu.

File: src/pages/Lend/Filter/FilterMenuSection.vue [hunk 3]
```diff
@@ -41,15 +50,36 @@ export default {
 @import 'settings';
 
 .filter-menu-section {
+	$filter-transition-timing: 0.15s ease-in;
```
Comment (ksekhri) [inline src/pages/Lend/Filter/FilterMenuSection.vue]: These feel so good.

**Comment by ksekhri** [comment]
@mcstover cool just pushed the latest cleanup, ready to review

File: src/pages/Lend/Filter/FilterMenuSection.vue (hunk 3 above)
Comment (mcstover) [reply src/pages/Lend/Filter/FilterMenuSection.vue]: 👍

File: src/pages/Lend/Filter/FilterMenuSection.vue (hunk 2 above)
Comment (mcstover) [reply src/pages/Lend/Filter/FilterMenuSection.vue]: This is fine. Just wondering if @click.stop="" would have worked too?

Comment (ksekhri) [reply src/pages/Lend/Filter/FilterMenuSection.vue]: Nice! It does and we can delete 3 lines of code :+1:

**Pull request merged**